app/views/user/attendances/show.html.erb

```diff
@@ -1,8 +1,11 @@
 <h1><%= link_to @module.name, turing_module_path(@module) %></h1>
+<%= link_to "Delete Attendance", attendance_path(@attendance), class: 's-button danger', data: { turbo: true, turbo_method: :delete, turbo_confirm: "WARNING!! \n\n Are you sure you want to delete this attendance?" } %>
```

Comment:
Did you test this locally? I think Turbo might be disabled and you'd have to enable it to get the confirmation alert.